src/build/do_fetch_litecore.ps1

```diff
@@ -68,11 +49,8 @@ foreach($arch in @("x86", "x86_64", "armeabi-v7a", "arm64-v8a")) {
     }
 }
 
-# Process Windows Libraries
-foreach($arch in @("x64", "x64_store", "arm64_store")) {
-	if(Test-Path $arch){
-		Remove-item $arch -Recurse
-	}
+if(Test-Path linux){
+    Remove-Item linux -Recurse
```

Comment:
yes

Reply:
This is to remove an existing linux folder if it exists from a previous run for some reason (this is where the new ones will be downloaded, and the final products are moved out of this folder).  It does not replace anything, though you might be noticing a deletion regarding x64, etc, which is for Windows.  Basically this script was too aggressive about deleting previous entries so it would wind up deleting things if you chose a subset of the download platforms (not an issue on Jenkins where it always downloads the full set, but annoying locally)